src/app/services/handlers/security-handler.service.ts

```diff
@@ -320,22 +319,22 @@ export class SecurityHandlerService {
   }
 
   elementAccess(element) {
-    if (element.domainType === DOMAIN_TYPE.DataModel ||
-      element.domainType === DOMAIN_TYPE.Terminology ||
-      element.domainType === DOMAIN_TYPE.CodeSet ||
-      element.domainType === DOMAIN_TYPE.ReferenceDataModel) {
+    if (element.domainType === CatalogueItemDomainType.DataModel ||
+      element.domainType === CatalogueItemDomainType.Terminology ||
+      element.domainType === CatalogueItemDomainType.CodeSet ||
+      element.domainType === CatalogueItemDomainType.ReferenceDataModel) {
```

Comment:
I like this, starting to phase out types that should belong to `mdm-resources` for other clients to use.